authors/apps/articles/migrations/0004_auto_20190419_2042.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.1.7 on 2019-04-19 17:42
```

Comment:
Please delete the three migration files you have and run `python manage.py makemigrations` to generate one file for your changes.

Reply:
I have pushed the changes please check them out